dataflow/manual/content.tex

```diff
@@ -299,6 +299,26 @@ \subsubsection{AbstractValue}
 class NullnessValue extends CFAbstractValue<NullnessValue>
 \end{verbatim}
 
+\subsubsection{UnusedAbstractValue}
+\label{sec:UnusedAbstractValue_classes}
+
+UnusedAbstractValue, as the name suggests, is an AbstractValue but is
+not involved in any computation during dataflow analysis. That is, the
+UnusedAbstractValue implements the leastUpperBound method in a way it
+just throws an error if it is called, without computing the lub with the
```

Comment:
The rest of this manual doesn't use `lub`. Please expand.

Reply:
done.